pages/[username]/galleries/[galleryId].tsx

```diff
@@ -35,7 +36,8 @@ export default function FocusedGallery({ galleryId, username }: Props) {
     <GalleryRoute
       navbar={
         <GalleryNavbar
-          galleryName={query.galleryById?.name || undefined}
+          showGalleryNameBreadcrumb
+          galleryRef={query.galleryById}
```

Comment:
instead of just passing the gallery name if we want to show it, we now have `showGalleryNameBreadcrumb` as a flag for the component.

We're always going to pass a gallery ref if we have one, so ^^ is an alternate way to flip the name on / off for `/username` vs `/username/galleries/galleryId`